<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProducerWidget</class>
 <widget class="QWidget" name="ImageProducerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filenameLabel">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Comments:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notesTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string>x</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Duration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="TimeSpinBox" name="durationSpinBox">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="defaultDurationButton">
         <property name="toolTip">
          <string>Make the current duration value the default value</string>
         </property>
         <property name="text">
          <string>Set Default</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="aspectRatioLabel">
       <property name="text">
        <string>Pixel aspect ratio</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="aspectNumSpinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="aspectDenSpinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="sequenceCheckBox">
       <property name="text">
        <string>Image sequence</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="repeatSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> frames</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>per picture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="proxyButton">
       <property name="text">
        <string>Proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuButton">
       <property name="toolTip">
        <string>Display a menu of additional actions</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="show-menu" resource="../../icons/resources.qrc">
         <normaloff>:/icons/oxygen/32x32/actions/show-menu.png</normaloff>:/icons/oxygen/32x32/actions/show-menu.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionCopyFullFilePath">
   <property name="text">
    <string>Copy Full File Path</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Show in Folder</string>
   </property>
  </action>
  <action name="actionSetFileDate">
   <property name="text">
    <string>Set Creation Time...</string>
   </property>
  </action>
  <action name="actionDisableProxy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable Proxy</string>
   </property>
  </action>
  <action name="actionMakeProxy">
   <property name="text">
    <string>Make Proxy</string>
   </property>
  </action>
  <action name="actionDeleteProxy">
   <property name="text">
    <string>Delete Proxy</string>
   </property>
  </action>
  <action name="actionCopyHashCode">
   <property name="text">
    <string>Copy Hash Code</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/timespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filenameLabel</tabstop>
  <tabstop>notesTextEdit</tabstop>
  <tabstop>defaultDurationButton</tabstop>
  <tabstop>aspectNumSpinBox</tabstop>
  <tabstop>aspectDenSpinBox</tabstop>
  <tabstop>sequenceCheckBox</tabstop>
  <tabstop>repeatSpinBox</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>menuButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
